### ISSUE TITLE
Deduplicating more things in descriptions

### ISSUE THREAD

**Issue by NetSysFire** [CleverRaven/Cataclysm-DDA#55821]
### Is your feature request related to a problem? Please describe.

Unfortunately, there are many repeated and non-unique redundant parts of item, terrain and similar descriptions. However, not all things may need a solution here since it may prove to be too complex for something as simple as a part of an item description re-used in a couple of items.

### Solution you would like.

It all started with #55207. There was also https://discourse.cataclysmdda.org/t/what-can-we-do-with-this-sentence-you-can-wear-it-if-you-like-but-it-wont-provide-any-effects/27548

List (not complete yet):

- [x] "Can be cut down with the right tools" (#55207)
- [ ] "Sterilized using radiation, so it's safe to eat." (low priority)
- [ ] "You could <harvest> if you examined <thing> more closely"
- [x] "You can wear it if you like, but it won’t provide any effects" (#55550)
- [ ] "Being caseless rounds, they cannot be disassembled or reloaded"
- [ ] "This one has been hand-reloaded"
- [ ] "Better throw it!" (explosive item warning)
- [ ] "You need a syringe to inject it… if you really want to?"
- [ ] "Seeing this is a bug"/"Dummy item" (low priority)

### Additional context

Do comment if you find more things, although as of writing I am not done going through the list of all items, or have any other feedback.
After collecting more strings which should be deduplicated, there should also be a quick search to determine how often the redundant string is used.

**Comment by LeahLuong** [comment]
Would #55521 apply to this issue?? Seems like the 2-sentence notify in the screenshot there could be streamlined to 1 if not removed altogether.

**Comment by haveric** [comment]
I'm planning on working on these plus a few others I've noted down from discord, but am waiting on at least one PR to get merged in order to avoid doing this more than once.

> Would #55521 apply to this issue?? Seems like the 2-sentence notify in the screenshot there could be streamlined to 1 if not removed altogether.

This is an entirely unrelated issue.

**Comment by NetSysFire** [comment]
Mind sharing those you found? I bet there is a whole lot more and if we collect those first there do not need to be 5 small PRs but maybe 1 big one if that makes sense

**Comment by haveric** [comment]
I didn't have a lot of time when I first responded, but here's the entire list combined with yours:
```
Sterilized using radiation, so it's safe to eat. (69 results)
This is a bug if you have it. (44 results)
You need a syringe to inject it and prime your body for mutation. (31 results)
If exposed to the atmosphere, it will start to go bad. (29 results)
Being caseless rounds, these cannot be disassembled or reloaded. (18 results)
Seeing this is a bug (18 results)
Protects from the elements as well as from harm. (16 results) (this one might be tricky due to it only applying to some suits, but I can figure something out)
This one has been hand-reloaded. (8 results)
Better throw it! (9 results)
---
if you really want to? (only used twice and doesn't really fit well with the others)
"Dummy item" isn't used repetitively from what I'm seeing.
"If you examined", "more closely, you could probably find" (none of these fit well with easy replacements unfortunately)
```

I'm waiting until https://github.com/CleverRaven/Cataclysm-DDA/pull/55776 gets merged to catch the new irradiated messages since it is the biggest one on the list, but I had fully intended to do as many of these as makes sense within a single PR.

**Comment by NetSysFire** [comment]
>Better throw it!

There are more variations of that. Check the item descriptions of armed explosives. E.g "Throw it immediately!", "It'll explode any moment now" and possibly more.

>"Dummy item" isn't used repetitively from what I'm seeing.

The tricky thing is that I combined this with "Seeing this is a bug", classifying both of them as "If you see this item, you should report this". There are also more varations of this, e.g "If you see this, something went wrong"/"this is a pseudo item". Some are also not even marked in this way, like all the basecamp items (basecamp charcoal smoker etc).

**Comment by haveric** [comment]
I've updated my list with the explosive ones. Let me know if you come across any more.

> The tricky thing is that I combined this with "Seeing this is a bug", classifying both of them as "If you see this item, you should report this". There are also more varations of this, e.g "If you see this, something went wrong"/"this is a pseudo item". Some are also not even marked in this way, like all the basecamp items (basecamp charcoal smoker etc).

I think these are going to end up being a different issue/PR, just due to the inconsistencies that I'm seeing.  They aren't as easy to drop in a replacement, but will need to be evaluated for consolidation/rewording.  It's still useful to include these though, so please keep them coming.

**Comment by Faalagorn** [comment]
I'd love to see "Seeing this is a bug" being dealt with, especially as a lot of items have it worded differently and it's not often clear if it's an actual item or a debug one, e.g. when testing game features.

**Comment by Faalagorn** [comment]
Also not essentially related, but when doing #55848 I found that a lot of effects have a "This is a bug if you have it." string in the description.

**Comment by Faalagorn** [comment]
Found another one: `generic item template`.
![screenshot_2022-03-06-133403](https://user-images.githubusercontent.com/4951331/156923475-4d6af158-2b10-46aa-b6e3-fdab09f63be0.png)
although it appears just one in Transifex, since the string is the same.
![screenshot_2022-03-06-133648](https://user-images.githubusercontent.com/4951331/156923552-0a307a94-04eb-4a4e-a187-23ef66687b30.png)
It may be worth evaluating combining with "seeing this is a bug" if relevant, though.

**Comment by haveric** [comment]
> I'd love to see "Seeing this is a bug" being dealt with

I didn't mean to imply that it wouldn't be taken care of, just that it's an outlier to the rest of these that will take more time to hunt down and consolidate everything into one (or more) consistent strings that can then be turned into flags hopefully.

**Comment by Faalagorn** [comment]
Another one is `this a pseudo item`:
![screenshot_2022-03-06-172638](https://user-images.githubusercontent.com/4951331/156932175-18c773d1-8954-49ab-a5d3-d04ec30873f4.png)

And @haveric yeah, I understand it will take time, in no way it's a urge to rush it, just voicing the need for that. :)

**Comment by Faalagorn** [comment]
`try throwing it instead`
![screenshot_2022-03-06-175643](https://user-images.githubusercontent.com/4951331/156933367-f7aabd92-5c80-47d3-a3d8-d4cec0afff52.png)

**Comment by Faalagorn** [comment]
It would be nice if the "Use it to XXX", "Activate to YYY" or possibly other variations could somehow be deduplicated/standardized, as they appear quite often in description, though I guess that would require a different approach that simply using flags.

**Comment by NetSysFire** [comment]
I think one relatively simple (at least I think) change would be to deduplicate all the leading whitespace in the epilogue messages. As far as I know there are a quite a bunch of epilogue messages distributed over some files, so it would at least help a tiny bit.

https://github.com/CleverRaven/Cataclysm-DDA/blob/597b469b0a464690f71f2b454133069a635c53e1/data/json/snippets/epilogue_factions.json#L6-L22

**Comment by Faalagorn** [comment]
As mentioned in #57167, there's currently 7 instances of appliances that have the "Ready to be plugged in." description + 1 "Ready to be plugged in and played.".

There's also "Can be plugged into a power source to start freezing food." for freezer, "can be plugged into the wall and used." for water purified, and several (9 it seems, unless there's some slight variations of it of "Can be placed and plugged into a power source.".

Also @NetSysFire some of the initial ones are fixed so they could be marked (sterilized using radiation; caseless rounds and sterilized using injection in #56361 by @haveric, so might as well mark them as done :)

**Comment by Faalagorn** [comment]
> I didn't have a lot of time when I first responded, but here's the entire list combined with yours:
> 
> ```
> Sterilized using radiation, so it's safe to eat. (69 results)
> This is a bug if you have it. (44 results)
> You need a syringe to inject it and prime your body for mutation. (31 results)
> If exposed to the atmosphere, it will start to go bad. (29 results)
> Being caseless rounds, these cannot be disassembled or reloaded. (18 results)
> Seeing this is a bug (18 results)
> Protects from the elements as well as from harm. (16 results) (this one might be tricky due to it only applying to some suits, but I can figure something out)
> This one has been hand-reloaded. (8 results)
> ---
> Better throw it! (9 results)
> It'll explode any moment now. (3 results)
> Throw it immediately! (2 results)
> ---
> if you really want to? (only used twice and doesn't really fit well with the others)
> "Dummy item" isn't used repetitively from what I'm seeing.
> "If you examined", "more closely, you could probably find" (none of these fit well with easy replacements unfortunately)
> ```
> 
> I'm waiting until #55776 gets merged to catch the new irradiated messages since it is the biggest one on the list, but I had fully intended to do as many of these as makes sense within a single PR.

Seeing how most of them are dealt with, it would be good to start nailing down the rest to close this PR, hopefully in time for 0.G.

I'm not sure if adding flags for them is a way to go (at least not for all), though apparently #56302 PR by @bombasticSlacks snuck some code "so that a variant can just append a description to the main description instead of having to retranslate the same thing over and over again" according to the wording at Discord, not sure if this can be used as is, or maybe some other code could be used to append the strings similar to how we do it with the vehicle part descriptions? That isn't 100% ideal, as there can be grammatical gender for items in some languages (e.g. "Use it to loosen" could be translated as "Użyj **go**, by **go** poluzować" or "Użyj **jej**, by **ją** poluzować, though it could be simplified to "Użyj, by poluzować", so it's all to be worked around with).

Anyway, first thing would be to find and then hopefully standardize all the description describing the same stuff. I'll try to nail them all down:

Unobtainable stuff, "This is a bug if you have it." seems to be the most fitting, but there are other variants of this:
* `This is a bug if you have it.` (4**3** results) – used for AI tags (one description mention it as a "Monster effect" though that could probably be changed to "AI tag" in `data/json/effects.json` and one in `data/json/npcs/Backgrounds/religious_1.json`
* `seeing this is a bug`/`Seeing this is a bug.  If seen, please report and destroy.`: (3 names + 15 descriptions)
* `If found in a game, it is a bug.`: (2 results)
* `Seeing this item is a bug.`:  (3 results)
* `IF YOU SEE THIS IT IS A BUG`: (9 results)
* `IF YOU ARE SEEING THIS IT IS A BUG` (2 results)
* `It is a bug if you have it.` (2 results) + 1 result of `It is a bug (heh) if you have it.` (which is a reference to having a bug mutation
* `if you have it something is wrong` (11 results)
* `Dummy item.  If you see this, then something went wrong.` (2 results)
* `if you see this, something went wrong` (1 result)
* `If you see this, it's a bug` (3 results)
* `Please report this bug.` (4 results)
* `this is a pseudo item` (7 results, but one is shared with `If you see this, it's a bug`)
* There are also generic items I mentioned in https://github.com/CleverRaven/Cataclysm-DDA/issues/55821#issuecomment-1059955437
I could unify this descriptions for a starters, especially as the "This is a crafting_pseudo_item if you have it something is wrong" are also Transifex issues.

Protects from the elements:
* `Protects from the elements` (indeed 16 elements, you mention that despite it being tricky, you can work something out @haveric)

Lit grenades:
* `Better throw it!` (9 results)
* `It'll explode any moment now.` (3 results)
* `Throw it immediately!` (2 results)
* `Throw it before it blows in your face!` (8 results)
* `Throw it quickly before it explodes.` (1 result)
* `It may be a good idea to throw it!` (1 result)
* `Throw it!` (1 result)
* `throwing it would be a good idea` (12 results, not sure if it's for lighted items)

Reloaded ammo:
* `This one has been hand-reloaded` (**9** results, one is in the middle of the sentence)
* `This one has been hand-loaded` (4 results, technically **re**loading is slightly different term than (hand)loading, but I don't know how much of a difference there is at a game)

Plug in appliances:
* `Ready to be plugged in.` (10 results, including one `Ready to be plugged in and played.`)
* `Can be plugged` (4 results)
* `Can be placed and plugged` (9 results)
* `You can plug` (1 result)

Activate to / use to:
* This is a little more complicated; there are many variations of this, though at least these could be unified.

**Comment by bombasticSlacks** [comment]
Without looking too closely. 

The variant stuff I added basically takes a base description "this is a thing. It has these properties" and if you add a variant the variant would be something like "blue thing" and the description would be "this particular one is navy blue." So in game a "blue thing" would have the description:

"this is a thing. It has these properties. this particular one is navy blue."

**Comment by Faalagorn** [comment]
> Without looking too closely.
> 
> The variant stuff I added basically takes a base description "this is a thing. It has these properties" and if you add a variant the variant would be something like "blue thing" and the description would be "this particular one is navy blue." So in game a "blue thing" would have the description:
> 
> "this is a thing. It has these properties. this particular one is navy blue."

It would work in that case, except these are not variants. I didn't look at your code, but if you could stitch some of the mentioned descriptions in a similar fashion that would surely improve both consistency and translators' work.
